core-concepts/streams/stream-names.md

```diff
@@ -25,9 +25,9 @@ A _category_ stream name does not have an ID. For example, the stream name for t
 
 ## Category Types
 
-Category stream names can be broken down into sub-categories, or _types_.
+Stream names can also have an optional list of _category types_.  
 
-The `command` type is a very common category type. It's used to indicate a stream that is used to transport _commands_ rather than _events_.
+One of the more common types associated with a category stream is the `command` type. It is used to indicate a stream that is used to transport _commands_ rather than _events_.
```

Comment:
I'm not inclined to the use of passive voice here. Also, the replacement of the contraction with its expansion increases formality, but I'm not sure what purpose is served.